titration/ui_state/user_value/pump_volume.py

```diff
@@ -13,7 +13,8 @@ def save_value(self, value):
         """
         The function to save the titrator's pump volume
         """
-        self.titrator.pump_volume = value
+        if value not in ("", "."):
```

Comment:
What does this statement do?

Reply:
It ensures that the value entered in the user_value states is an actual number.

Reply:
This is fixed in #115